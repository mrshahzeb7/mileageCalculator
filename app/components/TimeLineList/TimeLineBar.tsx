import React from 'react';
import {StyleSheet, View} from 'react-native';
import {colors} from '../../theme';
import {responsiveHeight, responsiveWidth} from '../../utils';

interface Props {}

export const TimeLineBar: React.FC<Props> = () => {
  return <View style={styles.line} />;
};

// styles
const styles = StyleSheet.create({
  line: {
    position: 'absolute',
    left: responsiveWidth(7),
    backgroundColor: colors.primaryButton,
    width: responsiveWidth(1),
    height: responsiveHeight(100),
  },
});
