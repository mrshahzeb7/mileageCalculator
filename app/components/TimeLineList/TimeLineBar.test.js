import 'react-native';
import React from 'react';
import {TimeLineBar} from './TimeLineBar';
import renderer from 'react-test-renderer';

describe('TimeLineBar', () => {
  it(' TimeLineBar renders correctly', () => {
    const tree = renderer.create(<TimeLineBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
