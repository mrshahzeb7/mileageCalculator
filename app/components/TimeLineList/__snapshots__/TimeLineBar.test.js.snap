// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeLineBar  TimeLineBar renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#93b2fd",
      "height": 1334,
      "left": 52.50000000000001,
      "position": "absolute",
      "width": 7.5,
    }
  }
/>
`;
